src/subscription_manager/cache.py

```diff
@@ -1112,3 +1064,51 @@ def _load_data(self, open_file):
         except ValueError:
             # Ignore json file parse error
             pass
+
+
+class ContentAccessModeCache(ConsumerCache):
```

Comment:
Hmm. It looks like that this class is already present in the `main` branch. This only moves it to another place.

Reply:
It was - I was trying to make sure the classes were all in the same order - and I think I was looking at another branch when I made this change.  I put it back to be on line 980 and re-checked all the classes in cache.py and everything looks ok now.